packages/ui-inputs/src/search-input.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react'
+import { useEffect, useState } from 'react'
+import { InputElement, InputRef, TextInput } from './text-input';
+import { Plus24Icon, Search24Icon } from '@kaltura-path/ui-icons';
+import { Theme } from './theme/theme';
+import { createUseStyles, theming } from './theme';
+
+const classNames = require('classnames');
+
+export interface SearchInputFieldProps {
+    /**
+     * Initial value for controlled input
+     * @default undefined
+     * */
+    value?: string;
+    /**
+     * Initial value for uncontrolled input
+     * @default undefined
+     * */
+    defaultValue?: string;
+    /**
+     * Set input in disabled state which prevents user input and changes input style
+     * @default undefined
```

Comment:
the default is false
```suggestion
     * @default false
```